golang/src/com/google/idea/blaze/golang/resolve/BlazeGoPackage.java

```diff
@@ -58,6 +58,7 @@
 import com.intellij.psi.PsiManager;
 import com.intellij.util.Processor;
 import java.io.File;
+import java.io.IOException;
```

Comment:
This is an unused import. Please run google-java-format on future PRs. Having to run google-java-format myself means that I need an additional manual step during the import of such PRs.

Reply:
My bad, I installed the formatter and ran it but it seems it doesn't optimize imports. Fixed in 
1ea0f3a